library/Rules/Cpf.php

```diff
@@ -13,8 +13,19 @@
 
 namespace Respect\Validation\Rules;
 
-class Cpf extends AbstractRule
+/**
+ * @author Alexandre Gaigalas <alexandre@gaigalas.net>
```

Comment:
Could you add `Validates a Brazillian CPF number.` as a description for this class?

Reply:
for sure

Reply:
And fix the name of Alexandre too.